src/main/java/core/basesyntax/dao/FruitsDao.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.dao;
+
+public interface FruitsDao {
+    boolean add(String[] fruits);
+
+    boolean remove(String[] fruits);
+}
```

Comment:
IMPORTANT: Refactor on your opinion, optional!!!

not bad idea, but mentors were said that we have anothe logic of this class: such as has 3 field (type of operation, name, 
quantity) and no have any methods.

Reply:
i used DAO